README.md

```diff
@@ -181,6 +182,225 @@ This section features documentation specifically designed for advanced users, ou
 - How to [trigger a reorg](docs/trigger-a-reorg/trigger-a-reorg.md).
 - How to [deploy contracts with the deterministic deployment proxy](docs/deterministic-deployment-proxy.md).
 
+## FAQ
+
+### Q: What are the different ways to deploy this package?
+
+<details>
+<summary><b>Click to expand</b></summary>
+
+1. Deploy the package without cloning the repository.
+
+```bash
+kurtosis run --enclave cdk github.com/0xPolygon/kurtosis-cdk
+kurtosis run --enclave cdk github.com/0xPolygon/kurtosis-cdk@main
+kurtosis run --enclave cdk github.com/0xPolygon/kurtosis-cdk@v0.2.15
+```
+
+2. Deploy the package with the default parameters.
+
+```bash
+kurtosis run --enclave cdk .
+```
+
+3. Deploy with the default parameters and specify on-the-fly custom arguments.
+
+```bash
+kurtosis run --enclave cdk . '{"deployment_stages": {"deploy_l1": false}}'
+```
+
+4. Deploy with a configuration file.
+
+```bash
+kurtosis run --enclave cdk --args-file params.yml .
```

Comment:
do we provide a sample params.yml anywhere for user reference ?

Reply:
I added a note to check the `tests` folder where most of our config files live.

[docs: link to sample config files](https://github.com/0xPolygon/kurtosis-cdk/pull/346/commits/ac4b7aa63e45cd6c138392c6d83529ac0aebc26b)